pom.xml

```diff
@@ -105,7 +105,7 @@
         <logback.version>1.2.3</logback.version>
         <slf4j.version>1.7.25</slf4j.version>  <!-- used for java.util.logging jul-to-slf4j interception -->
         <!-- Must match jclouds' version. From jclouds 1.9.3+ can be any version in the range [16-20) -->
-        <guava.version>18.0</guava.version>
+        <guava.version>19.0</guava.version>
```

Comment:
comment about helm needing v19 minimum (if that is the case)

Reply:
Yeap, all dependency upgrades i the root po were caued because Microbean Helm requires them.  :(